os/arch/arm/src/armv7-r/arm_dataabort.c

```diff
@@ -107,7 +111,13 @@ uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
 
 	/* Crash -- possibly showing diagnostic debug information. */
 
-	lldbg("\nData abort. PC: %08x DFAR: %08x DFSR: %08x\n", regs[REG_PC], dfar, dfsr);
+#ifdef CONFIG_SECURITY_LEVEL
+	if (get_security_level() == LOW_SECURITY_LEVEL)
+#endif
+	{
+		lldbg("\nData abort. PC: %08x DFAR: %08x DFSR: %08x\n", regs[REG_PC], dfar, dfsr);
+	}
```

Comment:
This kind of codes at every assert messages look not good.
How about below?
1. define new definition like `assertdbg`.
2. when the security level is changed, you can remap the definition to lldbg function.

Reply:
Thank you, i chaged that